ChangeLog.d/make_assemble_changelog_enforce_line_length.txt

```diff
@@ -0,0 +1,2 @@
+Changes
+   * Make assemble_changelog.py script enforce 80 character line limit
```

Comment:
I don't think changes in maintenance scripts warrant a changelog entry. The changelog file is meant to inform users and shouldn't have too much noise.